llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -614,6 +614,61 @@ void VPBasicBlock::print(raw_ostream &O, const Twine &Indent,
   printSuccessors(O, Indent);
 }
 #endif
+static void cloneCFG(VPBlockBase *Entry,
+                     DenseMap<VPBlockBase *, VPBlockBase *> &Old2NewVPBlocks);
+
+static VPBlockBase *cloneVPB(VPBlockBase *BB) {
```

Comment:
Well, the term `clone()` refers to "w/o-remapping-operands" when applied to recipes - as a virtual method rather than a static cloneR() function - so may seem natural if applied as such to their containing VPBB's and regions. The exception is VPlan::clone() which also takes care of remapping the operands. To better indicate when operands are remapped and when not, perhaps `unmappedClone()` and `mappedClone()` could be used instead, or `clone()` for w/o remapping, and, say, `VPlan::duplicate()` for w/ remapping (or vise versa).

Reply:
Updated to move block cloning to `::clone()` (in line with recipes) and renamed `VPlan::clone` to `VPlan::duplicate`.